easybuild/easyconfigs/s/scikit-image/scikit-image-0.13.1-foss-2017b-Python-3.6.3.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'Bundle'
+
+name = 'scikit-image'
+version = '0.13.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://scikit-learn.org/stable/index.html'
+description = """Scikit-learn integrates machine learning algorithms in the tightly-knit scientific Python world,
+building upon numpy, scipy, and matplotlib. As a machine-learning module,
+it provides versatile tools for data mining and analysis in any field of science and engineering.
+It strives to be simple and efficient, accessible to everybody, and reusable in various contexts."""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+exts_defaultclass = 'PythonPackage'
+exts_filter = ("python -c 'import %(ext_name)s'", '')
+
+dependencies = [
+    ('Python', '3.6.3'),
+    ('Qhull', '2015.2'),
+    ('matplotlib', '2.1.1', versionsuffix),
+    ('Pillow', '4.3.0', versionsuffix),
+]
+
+exts_list = [
+    ('networkx', '1.11', {
+        'source_urls': ['https://pypi.python.org/packages/source/n/networkx'],
+        'checksums': ['0d0e70e10dfb47601cbb3425a00e03e2a2e97477be6f80638fef91d54dd1e4b8'],
+    }),
+    (name, '0.13.0', {
```

Comment:
shouldn't it be `'0.13.1'`, or better still, `version`?

Reply:
Oh snap, good catch!